### PR TITLE
Add com.github.nihui.waifu2x-ncnn-vulkan

### PR DESCRIPTION
**Pull request by sat3ll**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/nihui/waifu2x-ncnn-vulkan/issues/150
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

CC: @TheEvilSkeleton

### CODE REVIEW

File: com.github.nihui.waifu2x-ncnn-vulkan.yaml
Comment (sat3ll) [reply com.github.nihui.waifu2x-ncnn-vulkan.yaml]: changed

**Comment by hfiguiere** [comment]
bot, build com.github.nihui.waifu2x-ncnn-vulkan

**Comment by TheEvilSkeleton** [comment]
Looks like the upstream repo doesn't provide any suitable screenshot. We'll have to make the bot ignore the metadata file.

File: flathub.json
Comment (TheEvilSkeleton) [inline flathub.json:3]:
```suggestion
  "skip-icons-check": true,
  "skip-appstream-check": true
```

**Comment by hfiguiere** [comment]
It's a command line tool, so it's ok

**Comment by TheEvilSkeleton** [comment]
bot, build com.github.nihui.waifu2x-ncnn-vulkan

**Comment by TheEvilSkeleton** [comment]
Works for me!

**Comment by TheEvilSkeleton** [comment]
@sat3ll what do you think about asking the upstream maintainer to comaintain it with you?

**Review by TheEvilSkeleton — APPROVED** [review]

**Comment by barthalion** [comment]
bot, build com.github.nihui.waifu2x-ncnn-vulkan

**Comment by barthalion** [comment]
```
> flatpak run com.github.nihui.waifu2x-ncnn-vulkan -i bebop.webp -o bebop2.webp
[0 AMD RADV RENOIR (ACO)]  queueC=1[4]  queueG=0[1]  queueT=0[1]
[0 AMD RADV RENOIR (ACO)]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 AMD RADV RENOIR (ACO)]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
[0 AMD RADV RENOIR (ACO)]  subgroup=64  basic=1  vote=1  ballot=1  shuffle=1
decode image bebop.webp failed
```

I assume it's something with my potato AMD APU? Mind attaching an original and resized image just so I'm sure it works before merging?

**Comment by sat3ll** [comment]
> ```
> > flatpak run com.github.nihui.waifu2x-ncnn-vulkan -i bebop.webp -o bebop2.webp
> [0 AMD RADV RENOIR (ACO)]  queueC=1[4]  queueG=0[1]  queueT=0[1]
> [0 AMD RADV RENOIR (ACO)]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
> [0 AMD RADV RENOIR (ACO)]  fp16-p/s/a=1/1/1  int8-p/s/a=1/1/1
> [0 AMD RADV RENOIR (ACO)]  subgroup=64  basic=1  vote=1  ballot=1  shuffle=1
> decode image bebop.webp failed
> ```
> 
> I assume it's something with my potato AMD APU? Mind attaching an original and resized image just so I'm sure it works before merging?

Not related, it's just unable to read the file. `bebop.webp` needs to be under `xdg-pictures` or you need to pass the directory to `flatpak run --filesystem`.

**Comment by barthalion** [comment]
Ah, you're right. I'd change it to home or even host. CLI (not TUI) apps can't really assume what they can do unless they're API clients.

**Comment by barthalion** [comment]
bot, build com.github.nihui.waifu2x-ncnn-vulkan

**Pull request closed without merging**
